docker-cloud/builds/automated-testing.md

```diff
@@ -79,9 +79,9 @@ Docker repository, regardless of the Autotest settings.
 
     The following options are available:
 
-    * `Off`: no additional tests. Test commits only to branches that are using Autobuild to build and push images.
-    * `Source Repository`: test commits to all branches of the source code repository, regardless of their Autobuild setting.
-    * `Source Repository & External Pull Requests`: tests commits to all branches of the source code repository, including any pull requests opened against it.
+    * `Off`: No additional test builds. Tests only run if they're configured as hooks in an automated build.
+    * `Source repository`: Run a test build for any pull requests to branches that match a build rule, but only when the pull comes from the same source repository.
```

Comment:
This is to match the current UI. I can patch the labels when we actually get them changed.